docs/network-flow-visibility.md

```diff
@@ -610,3 +612,48 @@ and other Theia features, please refer to the
 **Starting with Antrea v1.7, support for the ELK Flow Collector has been removed.**
 Please consider using the [Grafana Flow Collector](#grafana-flow-collector-migrated)
 instead, which is actively maintained.
+
+## L7 Visibility
+
+In Addition to Layer 4 network visibility, Antrea adds Layer 7 network flow visibility
+enabling users to collect and visualize application-layer flow data in Theia.
+
+### Usage
+
+To achieve L7 (Layer 7) network flow visibility, the `L7Visibility` and `TrafficControl`
+feature gates must be enabled.
+
+User can visualize the L7 network visibility feature using annotations:
```

Comment:
"visualize" a feature doesn't mean much IMO. Do you mean "enable" instead?

Reply:
I meant to export the flows, in order to visualize on Theia.
changed it to 
```
To export layer 7 flows of a Pod or Namespace, user can annotate Pods or Namespaces
with the annotation key `visibility.antrea.io/enable-l7` and set the value to indicate
the traffic flow direction, which can be ingress, egress or both.
```